parachain/runtime/src/lib.rs

```diff
@@ -285,16 +284,19 @@ impl dummy_verifier::Trait for Runtime {
 	type Scheduler = scheduler::Module<Runtime>;
 }
 
-impl dummy_app::Trait for Runtime {
-	type Event = Event;
-}
-
 impl balances::Trait<balances::Instance2> for Runtime {
-	type Balance = u128; // We'll need to consider using a BigNum (u256) for PolkaETH
+	type Balance = u128; // We'll need to use a BigNum (U256) for PolkaETH
```

Comment:
Should we add a TODO here?

Reply:
ye or github issue

Reply:
There already is one from the last milestone: https://github.com/Snowfork/polkadot-ethereum/issues/18  :)